packages/types/src/lib/interfaces.ts

```diff
@@ -33,11 +33,36 @@ export interface AccsOperatorParams {
 
 /** ---------- Auth Sig ---------- */
 
+/**
+ * An `AuthSig` represents a cryptographic proof of ownership for an Ethereum address, created by signing a standardized [ERC-5573 SIWE](https://eips.ethereum.org/EIPS/eip-5573) (Sign-In with Ethereum) message. This signature serves as a verifiable credential, allowing the Lit network to associate specific permissions, access rights, and operational parameters with the signing Ethereum address. By incorporating various capabilities, resources, and parameters into the SIWE message before signing, the resulting `AuthSig` effectively defines and communicates these authorizations and specifications for the address within the Lit network.
```

Comment:
So technically `5573` is for `SIWE ReCap` see here: https://ethereum-magicians.org/t/eip-5573-siwe-recap/10627 

So while you are correct I think we should specify `SIWE ReCap` over just `SIWE`